ceph_iscsi_config/common.py

```diff
@@ -152,6 +153,18 @@ def _get_ceph_config(self):
 
         return cfg_dict
 
+    def init_config(self):
+        ioctx = self.ceph.cluster.open_ioctx(self.pool)
```

Comment:
Nit: probably should handle the case where the pool doesn't exist. Perhaps move the code from [1] to a common ```_open_ioctx``` function that can be used both places.

[1] https://github.com/vshankar/ceph-iscsi-config/blob/451e2779a332887dec347c213dd7580b86b16c09/ceph_iscsi_config/common.py#L127